packages/react-scripts/plop/reactComponent/index.js

```diff
@@ -10,6 +10,17 @@ module.exports = (plop) => {
       type: 'input',
       name: 'name',
       message: 'Name:',
+    }, {
+      type: 'list',
+      name: 'type',
+      message: 'Choose component type:',
+      choices: [{
+        name: 'Class component',
+        value: 'class',
+      },{
+        name: 'Functional component',
```

Comment:
Powinien nazywać się `function component`. Wiem, że brzmi dziwnie, ale już było o to gównoburza w całym community. Oficjalną wersją jest `function`. :)